analyzers/src/SonarAnalyzer.CSharp/Rules/FunctionComplexity.cs

```diff
@@ -27,26 +27,24 @@
 using SonarAnalyzer.Common;
 using SonarAnalyzer.Helpers;
 using SonarAnalyzer.Metrics.CSharp;
+using StyleCop.Analyzers.Lightup;
 
 namespace SonarAnalyzer.Rules.CSharp
 {
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
     [Rule(DiagnosticId)]
     public class FunctionComplexity : ParameterLoadingDiagnosticAnalyzer
```

Comment:
Please take a look at why it does not inherit from` FunctionComplexityBase`

Reply:
The following issue is created to tackle this. https://github.com/SonarSource/sonar-dotnet/issues/4675